examples/java/src/main/java/org/apache/beam/examples/common/ExampleOptions.java

```diff
@@ -17,14 +17,14 @@
  */
 package org.apache.beam.examples.common;
 
-import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
 import org.apache.beam.sdk.options.Default;
 import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
 
 /**
- * Options that can be used to configure the Dataflow examples.
+ * Options that can be used to configure the Beam examples.
  */
-public interface DataflowExampleOptions extends DataflowPipelineOptions {
+public interface ExampleOptions extends PipelineOptions {
   @Description("Whether to keep jobs running on the Dataflow service after local process exit")
```

Comment:
updated the description for now.

I am not removing references to this options, since it might be useful for other runners.